torch/fx/experimental/normalize.py

```diff
@@ -27,13 +28,15 @@ class NormalizeArgs(Transformer):
         traced = NormalizeArgs(traced).transform()
     """
     def __init__(self, module : torch.nn.Module, normalize_functionals : bool = True,
-                 normalize_modules : bool = True):
+                 normalize_modules : bool = True,
+                 normalize_to_only_use_kwargs : bool = True):
         super().__init__(module)
         self.normalize_functionals = normalize_functionals
         self.normalize_modules = normalize_modules
+        self.normalize_to_only_use_kwargs = normalize_to_only_use_kwargs
```

Comment:
I think this will still be broken for operators. Can you add a flag `self.normalize_operators` that defaults to False here?

Reply:
I'm a little confused on this one -- AFAICT `NormalizeArgs` and `NormalizeOperators` are totally independent. What would `self.normalize_operators` do inside `NormalizeArgs`?

Reply:
Friendly ping @jamesr66a 🙂

Reply:
The `else` branch starting at line 65 normalizes arguments for nodes with operator targets. `NormalizeOperators` changes operator targets, not args

Reply:
Got it, added.